Code/Framework/AzFramework/AzFramework/Viewport/CameraInput.cpp

```diff
@@ -275,7 +275,8 @@ namespace AzFramework
         {
             auto& cameraInput = m_idleCameraInputs[idleIndex];
             const bool canBegin = cameraInput->Beginning() &&
-                AZStd::all_of(m_activeCameraInputs.cbegin(), m_activeCameraInputs.cend(),
+                AZStd::all_of(m_activeCameraInputs.cbegin(),
+                              m_activeCameraInputs.cend(),
                               [](const auto& input)
                               {
                                   return !input->Exclusive();
```

Comment:
This is down to the change in .clang-format for `BinPackArguments: false` - the changes are pretty minor so I've kept them in this PR